plugins/destination/mssql/queries/definition.go

```diff
@@ -46,30 +46,11 @@ func (d *Definition) Nullable() *Definition {
 	}
 }
 
-func NewDefinition(name, typ string, nullable bool) *Definition {
-	d := &Definition{
-		Name:    strings.ToLower(name),
-		typ:     strings.ToLower(typ),
-		notNull: !nullable,
-	}
-	// add unique for _cq_id
-	d.unique = d.Name == schema.CqIDColumn.Name
```

Comment:
I'm not sure why this is needed. As a general approach we should not rely on CqID existing, and only do whatever is  specified in `column.CreationOptions`

Reply:
we had `unique not null` in PostgreSQL previously. If we skip this now, that's fine